carma-messenger-core/cpp_message/test/test_MobilityPath.cpp

```diff
@@ -69,10 +87,11 @@ TEST(MobilityOperationMessageTest, testEncodeMobilityOperationMsg)
     message.trajectory=trajectory;
     auto res = worker.encode_mobility_path_message(message);
     
-   // std::vector<uint8_t> binary_input={0,242,112,77,90,113,39,212,90,209,171,22,12,38,173,56,147,234,45,104,213,131,150,172,88,65,133,14,36,88,204,88,177,98,197,139,22,43,89,50,100,201,107,54,108,217,173,131,6,12,21,172,88,177,98,197,139,22,44,88,177,98,229,147,38,108,219,178,96,205,179,134,173,27,183,106,225,131,116,193,149,6,137,131,42,13,83,6,84,27,57,100,201,155,54,236,152,51,108,225,171,70,237,218,184,96,220,39,213,245,125,95,103,217,246};
-    std::vector<uint8_t> to_read=res.get();
-    auto len=to_read.size();
-    //for(auto i=0;i<len;i++)std::cout<<int(to_read[i])<<",";
+    // std::vector<uint8_t> to_read=res.get();
+    // size_t len=to_read.size();
```

Comment:
Same as above, is there no reason to validate the encoded binary matches the original binary?

Reply:
The binary array is being generated here by encoding a cav_msg::MobilityRequest input. The test was redundant here because I was using the output of the test to make the check